ui/ui-community/pages/imports/[creatorId].tsx

```diff
@@ -1,8 +1,14 @@
 import Head from "next/head";
 import ImportList from "@grouparoo/ui-components/components/import/List";
 import { useRouter } from "next/router";
+import { withServerErrorHandler } from "@grouparoo/ui-components/utils/withServerErrorHandler";
+import { NextPageWithInferredProps } from "@grouparoo/ui-components/utils/pageHelper";
 
-export default function Page(props) {
+export const getServerSideProps = withServerErrorHandler(async (ctx) => {
+  return { props: await ImportList.hydrate(ctx) };
+});
```

Comment:
The reason I like this at the top is that we are inferring the props, we can always go up to see what the server side props are returning instead of scrolling down, and it puts both the end of the function and the start of the page component together.